kubernetes/loculus/values.yaml

```diff
@@ -1212,10 +1211,13 @@ defaultOrganisms:
     <<: *defaultOrganismConfig
     preprocessing:
       - <<: *preprocessing
-        configFile:
-          <<: *preprocessingConfigFile
-          nextclade_dataset_server: https://raw.githubusercontent.com/nextstrain/nextclade_data/ebola/data_output
-          nextclade_dataset_name: nextstrain/ebola/sudan
+        version: 1
+        nextclade_dataset_name: nextstrain/ebola/sudan
+        nextclade_dataset_server: https://raw.githubusercontent.com/nextstrain/nextclade_data/ebola/data_output
+      - <<: *preprocessing # test having 2 simultaneous versions
+        version: 2
+        nextclade_dataset_name: nextstrain/ebola/sudan
+        nextclade_dataset_server: https://raw.githubusercontent.com/nextstrain/nextclade_data/ebola/data_output
```

Comment:
@corneliusroemer, @fhennig, this is breaking the dummy test organism because no lineage system definition is specified for pipelien version 2. This causes the SILO import job to throw "No URL defined for pipeline version 2"

Reply:
attempt at a fix: https://github.com/loculus-project/loculus/pull/3534/commits/49225145e159d12586bc4085da8f00504b443659

(But maybe we want to add better warnings/tests around this?)

Reply:
maybe the way the lineages are defined could also be changed to be put into the preprocessing config